File: fixtures/worker-thread.kt
suspend fun summarize(values: MutableList<Double>): Double {
  return kotlinx.coroutines.withContext(kotlinx.coroutines.Dispatchers.Default) {
    var total: Double = 0.0
    for (value in values) {
      total = (total + value)
    }
    return@withContext total
  }
}

suspend fun label(count: Double): String {
  return kotlinx.coroutines.withContext(kotlinx.coroutines.Dispatchers.Main) {
    return@withContext "count: " + lucentStr(count)
  }
}

File: fixtures/worker-thread.swift
func summarize(values: [Double]) async throws -> Double {
  return try await Task.detached {
    var total: Double = 0.0
    for value in values {
      total = total + value
    }
    return total
  }.value
}

@MainActor func label(count: Double) async throws -> String {
  return "count: " + lucentStr(count)
}
